content/ember-released-5-3.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Ember 5.3 Released
+authors:
+  - jen-weber
+  - katie-gengler
+date: 2023-09-23T00:00:00.000Z
+tags:
+  - releases
+  - '2023'
+  - version-5-x
+---
+
+Today the Ember project is releasing version 5.3 of Ember.js, EmberData, and Ember CLI. <!-- Block start: Uncomment if an LTS candidate --><!--This release of Ember.js is an LTS (Long Term Support) candidate. LTS candidates prioritize stability over the addition of new features, and have an extended support schedule.--><!-- Block end -->
+
+This release kicks off the 5.4 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 5.3
+
+Ember.js 5.3 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Deprecations
+
+Ember.js 5.3 introduced 1 deprecation.
+
+1. Deprecate implicit record loading in Ember Route per [RFC #774](https://rfcs.emberjs.com/id/0774-implicit-record-route-loading). Please see [the deprecation guide](https://deprecations.emberjs.com/v5.x#toc_deprecate-implicit-route-model) for more information.
+<!-- Block start: If there were no deprecations, remove this block -->
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+<!-- Block end -->
+
+For more details on changes in Ember.js 5.3, please review the [Ember.js 5.3.0 release page](https://github.com/emberjs/ember.js/releases/tag/v5.3.0).
+
+---
+
+## EmberData
+
+EmberData is the official data persistence library for Ember.js applications.
+
+### Changes in EmberData 5.3
+
+#### Bug Fixes
+
+EmberData 5.3 introduced 10 bug fixes. Details are in the
+[EmberData 5.3.0 release page](https://github.com/emberjs/data/releases/tag/v5.3.0).
+
+#### Features
+
+EmberData 5.3 introduced 3 features.
```

Comment:
```suggestion
EmberData 5.3 introduced several new features.
```

Reply:
The release notes was a pretty high level overview, if we want to get into brass tacks:

- the 3 cache methods
- 5 builders each for REST/ActiveRecord/JSON:API
- 2 serialization utils for JSON:API
- 6 builder utils
- a basic lifetimes service

Depending on how you're counting thats as many as 17 new features :D

Reply:
And a partridge in a pear tree